validation/pipeline/tfp/Jenkinsfile.airgap.tests

```diff
@@ -0,0 +1,180 @@
+          
+                withEnv(paramsMap) {
+
+
+                    stage('Checkout Infrastructure Repo') {
+                        deleteDir()
+                        checkout([
+                                    $class: 'GitSCM',
+                                    branches: [[name: "*/${infraBranch}"]],
+                                    extensions: scm.extensions + [[$class: 'CleanCheckout']],
+                                    userRemoteConfigs: infraRepo
+                                ])
+                     }
+
+                    stage('Configure and Build') {
+                        config = config.replace('${AWS_SECRET_ACCESS_KEY}', env.AWS_SECRET_ACCESS_KEY)
+                        config = config.replace('${AWS_ACCESS_KEY_ID}', env.AWS_ACCESS_KEY_ID)
+                
+                        writeFile file: filename, text: config
+                
+                        dir(".ssh") {
+                            def decoded = new String(env.AWS_SSH_PEM_KEY.decodeBase64())
+                            writeFile file: AWS_SSH_KEY_NAME, text: decoded
+                    
+                            def decodedRsa = new String(AWS_SSH_RSA_KEY.decodeBase64())
+                            writeFile file: JENKINS_RKE_VALIDATION, text: decodedRsa
+                        }
+                        
+                        env.CATTLE_TEST_CONFIG=rootPath+filename
+                
+                        sh "./configure.sh"
+                        sh "./build.sh"
+                     }
+
+                    stage('Run Infrastructure Module Test') {
+                        try {
+                            sh """
+                            docker run --name ${testContainer} -t --env-file ${envFile} ${imageName} sh -c "
+                            /root/go/bin/gotestsum --format standard-verbose --packages=${testsDir} --junitfile ${testResultsOut} --jsonfile ${testResultsJSON} -- -timeout=${timeout} -v ${params.TEST_CASE} | tee /root/test_logs.txt"
+                            """
+                            sh "docker cp ${testContainer}:/root/ ./"
+              
+                            adminToken = sh(script: "grep 'Admin bearer token:' ./root/test_logs.txt | awk -F 'Admin bearer token: ' '{print \$2}' | tr -d '\"'", returnStdout: true).trim()
+                            echo "Admin token: ${adminToken}"
+                            privateRegistry = sh(script: "grep 'Private registry:' ./root/test_logs.txt | awk -F 'Private registry: ' '{print \$2}' | tr -d '\"'", returnStdout: true).trim()
+                            echo "Private registry:  ${privateRegistry}"
+
+                        } catch(err) {
+                            echo 'Test run had failures. Collecting results...'
+                        }
+                     }
+
+                    stage('Checkout Test Repo') {
```

Comment:
nit, Tests repo sounds better (Test sounds like we're testing something in the file 😅 )